src/panels/lovelace/editor/config-elements/entity-selectors/hui-glance-entity-select.ts

```diff
@@ -0,0 +1,129 @@
+import { html, PropertyDeclarations } from "@polymer/lit-element";
+import { TemplateResult } from "lit-html";
+import "@polymer/paper-dropdown-menu/paper-dropdown-menu";
+import "@polymer/paper-item/paper-item";
+import "@polymer/paper-listbox/paper-listbox";
+
+import { HuiEntityEditor } from "../../../components/hui-entity-editor";
+import { HomeAssistant } from "../../../../../types";
+import { ConfigEntity } from "../../../cards/hui-glance-card";
+
+export class HuiGlanceEntitySelect extends HuiEntityEditor {
+  static get properties(): PropertyDeclarations {
+    return {
+      hass: {},
+      entities: {},
+    };
+  }
+  protected hass?: HomeAssistant;
+  protected entities?: ConfigEntity[];
+
+  protected renderOptions(
+    entityConf: ConfigEntity,
+    index: number
+  ): TemplateResult {
+    return html`
+      <paper-toggle-button>Options</paper-toggle-button>
+      <div class="options">
+        <paper-input
+          label="Name"
+          .value="${entityConf.name || ""}"
```

Comment:
Check how we did this at the editor: https://github.com/home-assistant/home-assistant-polymer/blob/dev/src/panels/lovelace/editor/config-elements/hui-glance-card-editor.ts#L60

Reply:
And then filter it: https://github.com/home-assistant/home-assistant-polymer/blob/dev/src/panels/lovelace/editor/config-elements/hui-glance-card-editor.ts#L107